user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/General/Maps/MapsTutorial.md

```diff
@@ -0,0 +1,121 @@
+
+![CellTowersTemplateResult](~/user-guide/images/MapsCellTowersTemplateResult.png)
+
+## Step 5: Add grouped markers
+
+The templates added in the previous step are looking good, but the visualization feels cluttered when zoomed out al the way. In this step we will add grouped markers. The catalog package from step 1 contains 2 more JSON files, *CellTowersGroups* & *CellTowersCities*, that represent a more grouped overview of the cell towers. To visualize these we need 2 new queries identical to our first one, but with a different file as a source. Theses queries can then be added to the maps component. We also configure a simple template for both queries that shows the *Count* of cell towers that are grouped in the markers.
+
+![GroupedCellTowersTemplate](~/user-guide/images/MapsGroupedCellTowersTemplate.png)
+
+We only want to show these new markers between certain zoom levels, this can be configured in the *Advanced settings* part of the layer settings.
+
+- Cell towers should be shown from 60 to 100.
+- Cell towers groups should be shown from 52 to 60.
+- Cell towers cities should be shown from 50 to 52.
+
+This configuration makes more and more grouped markers appear when zooming in on the map until the cell towers themselves become visible.
+
+![GroupedCellTowersTemplate](~/user-guide/images/MapsGroupedCellTowers.gif)
+
+## Step 6: Visualize connections
+
+Now that all the cell towers are visualized on the map, we can visualize the connections between them. The *CellTowers* dataset contains a *source* column with the ID of the cell tower an other tower is linked to. We can create a new query that starts from our *CellTowers* query and joins on itself based on this column. We also create a new concatenated column of the IDs of both towers and only select the interesting columns. This results in a new dataset with a unique ID, and 2 sets of coordinates that we can visualize as lines on the map.
+
+![CellTowersConnectionsQuery](~/user-guide/images/MapsCellTowersConnectionsQuery.png)
+
+We want to be able to show and hide these connections, so we will add them to the maps component in an other layer than the cell towers. This is done by first adding the query as before. Then, a new layer is added in the layer settings and the query can be dragged & dropped from the first layer onto that new layer. To make sure the connections are shown underneath our markers in the other layer, the layer weight can be set higher. By default, the query will be visualized as markers, this can be changed by hovering over the query name and clicking *Set as line*. The component will try to automatically configure the dimensions and show the lines on the map. Just as with the markers, lines can also by styled to match our application. 
+
+![CellTowerConnectionsSettings](~/user-guide/images/MapsCellTowersConnectionsSettings.png)
+
+![CellTowerConnections](~/user-guide/images/MapsCellTowersConnections.png)
```

Comment:
This image is missing in the pull request, which causes the build to fail. Could you add this or else remove this reference?